nodejs/eks/nodegroup.ts

```diff
@@ -1944,6 +1942,19 @@ function createMNGCustomLaunchTemplate(
     );
 }
 
+function getClusterServiceCidr(
+    networkConfig: pulumi.Output<aws.types.output.eks.ClusterKubernetesNetworkConfig>,
```

Comment:
Is there a way to know this earlier in the Input phase not Output? E.g. to throw a better preview? or doesn't matter.

Reply:
We cannot know it earlier because EKS sets the service CIDR unless the user overrides it.

But overall it doesn't matter. This is used only in the user data and that is fully dependent on multiple outputs of the cluster.